consensus/poet/cli/sawtooth_poet_cli/characteristic.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+from sawtooth_poet_cli import config
+from sawtooth_poet_cli.poet_enclave_module_wrapper import \
+    PoetEnclaveModuleWrapper
+
+
+def add_characteristic_parser(subparsers, parent_parser):
+    """Add argument parser arguments for the `poet characteristic` subcommand.
```

Comment:
@vaporos Are `poet enclave measurement` and `poet enclave basename` more palatable?